blt/src/Blt/Plugin/Commands/HsCommands.php

```diff
@@ -50,11 +53,41 @@ public function postCodeDeployUpdate() {
         $failed[] = $site;
       }
     }
+    unlink(sys_get_temp_dir() . '/update-report.txt');
+
     $this->yell(sprintf('Updated %s sites successfully.', count($success)), 100);
+
     if ($failed) {
       $this->yell(sprintf("Update failed for the following sites:\n%s", implode("\n", $failed)), 100, 'red');
+
+      if (EnvironmentDetector::isAhStageEnv() || EnvironmentDetector::isAhProdEnv()) {
+        $this->sendSlackNotification("A new deployment has been made to *$target_env* using *$deployed_tag*. At least one site failed updating.");
```

Comment:
Why not pass the failed sites? Would the message be too long if too many sites failed?

Reply:
Yeah I was kinda on the fence about it. I figured it might be less noise in Slack. Maybe we can use the number of sites that failed? then it's still short message

Reply:
Ooh I like that idea. A count would be useful without creating too much noise.